### PR TITLE
2565 bump wallet version

### PR DESCRIPTION
**Pull request by tnickelsen**

### CODE REVIEW

File: domains/certificates/Query.API/API.IntegrationTests/Factories/RegistryConnectorApplicationFactory.cs [hunk 1]
```diff
@@ -19,11 +19,10 @@ public class RegistryConnectorApplicationFactory : WebApplicationFactory<registr
     public string ConnectionString { get; set; } = "";
     public RabbitMqOptions? RabbitMqOptions { get; set; }
     public string OtlpReceiverEndpoint { get; private set; } = "http://foo";
-    public ProjectOriginOptions? ProjectOriginOptions { get; set; } = new()
+    public PoRegistryOptions? PoRegistryOptions { get; set; } = new()
```
Comment (TopSwagCode) [inline domains/certificates/Query.API/API.IntegrationTests/Factories/RegistryConnectorApplicationFactory.cs:22]: minor. Fullname: ProjectOriginRegistryOptions

File: domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/ModifyWalletUrlsOnContractsToContainWalletApiTests.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+namespace API.IntegrationTests.Shared.Migrations;
+
+[Collection(IntegrationTestCollection.CollectionName)]
+public class ModifyWalletUrlsOnContractsToContainWalletApiTests
+{
+    private readonly DbContextOptions<ApplicationDbContext> options;
+
+    public ModifyWalletUrlsOnContractsToContainWalletApiTests(IntegrationTestFixture integrationTestFixture)
+    {
+        var newDatabaseInfo = integrationTestFixture.PostgresContainer.CreateNewDatabase().Result;
```
Comment (TopSwagCode) [inline domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/ModifyWalletUrlsOnContractsToContainWalletApiTests.cs:23]: .Result <-- Could this cause problems on slow machines when not using .GetAwaiter().Result?

**Review by TopSwagCode — DISMISSED** [review]

File: domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/ModifyWalletUrlsOnContractsToContainWalletApiTests.cs (hunk 2 above)
Comment (tnickelsen) [reply domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/ModifyWalletUrlsOnContractsToContainWalletApiTests.cs:23]: we can't await in a constructor

File: domains/certificates/Query.API/API.IntegrationTests/Factories/RegistryConnectorApplicationFactory.cs (hunk 1 above)
Comment (tnickelsen) [reply domains/certificates/Query.API/API.IntegrationTests/Factories/RegistryConnectorApplicationFactory.cs]: renamed

**Review by TopSwagCode — APPROVED** [review]

**Pull request merged**